### PR TITLE
[FLINK-27544][docs]Example code in 'Structure of Table API and SQL Programs' is out of date and cannot run.

### PR DESCRIPTION
**Pull request by ChengkaiYang2022**
## What is the purpose of the change

Update outdated code in the example, for more details please see [FLINK-27544 on jira](https://issues.apache.org/jira/browse/FLINK-27544?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=17533584#comment-17533584).

## Brief change log

Edit example code in common.md in content\docs\dev\table and content.zh\docs\dev\table.

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no 
  - The serializers: (yes / no / don't know) no
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components),  Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) no

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by ChengkaiYang2022** [comment]
The test code is in my [github Repository(version 1.15)](https://github.com/ChengkaiYang2022/flink-test/blob/main/flink115/src/main/java/com/yck/TestTableAPI.java#L22) and [version 1.14](https://github.com/ChengkaiYang2022/flink-test/blob/main/flink114/src/main/java/com/yck/TestTableAPI.java) which are also mentioned in JIRA.

**Comment by ChengkaiYang2022** [comment]
Hi, @RocMarshal I found some outdated code in the document, would you mind take a look at this?For more details you can check the jira page

**Review by RocMarshal — APPROVED** [review]
Thanks for the contribution. 
LGTM +1.

**Comment by ChengkaiYang2022** [comment]
Could you @leonardBang  help to check it ?

**Comment by ChengkaiYang2022** [comment]
Hi,Could you @MartijnVisser  help to check it ?

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@ChengkaiYang2022 Thanks for this. I think this could subsume https://github.com/apache/flink/pull/19111 but it needs some more polishing.

File: docs/content/docs/dev/table/common.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ tableEnv.createTemporaryTable("SourceTable", TableDescriptor.forConnector("datag
...
-    .option(DataGenOptions.ROWS_PER_SECOND, 100)
+    .option(DataGenConnectorOptions.ROWS_PER_SECOND, 100L)
     .build());
 
 // Create a sink table (using SQL DDL)
-tableEnv.executeSql("CREATE TEMPORARY TABLE SinkTable WITH ('connector' = 'blackhole') LIKE SourceTable");
+tableEnv.executeSql("CREATE TEMPORARY TABLE SinkTable WITH ('connector' = 'blackhole') LIKE SourceTable (EXCLUDING OPTIONS) ");
 
 // Create a Table object from a Table API query
 Table table2 = tableEnv.from("SourceTable");
```
[the remark this answers is not in the document]
Comment (ChengkaiYang2022) [reply docs/content/docs/dev/table/common.md]: @MartijnVisser Good catch! Fixed.

**Review by MartijnVisser — APPROVED** [review]
@ChengkaiYang2022 Thanks for this! LGTM

**Pull request merged**